README.md

```diff
@@ -149,9 +149,18 @@ Notes for developers
 
 To start contributing to Mosh, install the following dependencies:
 
+Debian, Windows Subsystem for Linux:
+
+```
+$ sudo apt install -y build-essential debhelper-compat protobuf-compiler \
+    libprotobuf-dev pkg-config libutempter-dev zlib1g-dev libncurses5-dev \
+    libssl-dev bash-completion locales tmux less
+```
+
+MacOS:
+
 ```
-$ sudo apt install -y protobuf-compiler libprotobuf-dev libutempter-dev autoconf automake nettle-dev
-$ brew install protobuf automake
+$ brew install protobuf automake zlib openssl bash tmux less
```

Comment:
I would limit this to what we have in the CI for now

Reply:
Done